SeleniumWrapper.Core/BrowserConfiguration/BrowserService.cs

```diff
@@ -0,0 +1,58 @@
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumWrapper.Core.BrowserConfiguration;
+
+public static class BrowserService
+{
+    private static BrowserModel BrowserModel { get; set; }
+    private static readonly ThreadLocal<IBrowser> BrowserContainer = new ThreadLocal<IBrowser>();
+    
+    private static readonly ThreadLocal<BrowserFactory> BrowserFactoryContainer = new ThreadLocal<BrowserFactory>();
+
+    private static bool IsApplicationStarted() => BrowserContainer.IsValueCreated && BrowserContainer.Value.IsStarted;
+
+    public static Browser StartBrowser(BrowserModel browserModel)
```

Comment:
StartBrowser takes care only for starting browser, not for initializing another instances.Single responsibility principle is broken